build.sh

```
@@ -67,27 +80,27 @@ cd ..
     echo "✅ Build successful: $ZIP_FILE created"
     echo "File path: $(pwd)/$ZIP_FILE"
-    
+
     # Deploy to local WordPress installation if environment variable is set
     if [ -n "$WP_LOCAL_PLUGIN_DIR" ]; then
         echo "\nDeploying to local WordPress installation..."
         echo "Deploying to local WordPress installation..."
-        
+
         # Remove existing plugin directory
         rm -rf "$WP_LOCAL_PLUGIN_DIR/$PLUGIN_SLUG"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)